opm/parser/eclipse/EclipseState/Schedule/Group/Group.hpp

```diff
@@ -133,6 +134,7 @@ struct GroupInjectionProperties {
         serializer(reinj_group);
         serializer(voidage_group);
         serializer(injection_controls);
+        serializer(available_group_control);
```

Comment:
I'm sure it does not matter, but the order is different between the serializer calls and the data members.